crates/lib3h_protocol/src/protocol.rs

```diff
@@ -58,10 +58,14 @@ pub enum ClientToLib3hResponse {
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub enum Lib3hToClient {
     // -- Connection -- //
+    /// Notification of successful connection to a network
+    Connected(ConnectedData),
```

Comment:
This is a response to `ClientToLib3h::Connect`, we don't need to add it here, it's already `ClientToLib3hResponse::ConnectResult`

Reply:
Its not a response to a Connect since `Connect` is a request to connect to a specific peer.
`Connected` is a notification that we are connected to the **network**.

Reply:
... as in the network health stuff that gives us partition warnings, etc? I was assuming we weren't implementing any of that, since we haven't fleshed out the details of the core/lib3h protocol for it.

Reply:
Since, in the end, we are not going to have a "Connect" request in the first place I figured the only use for this would be for the network health thing yes.

Reply:
Maybe we need to rename this. I think we're going to still need a "Bootstrap" message, i.e. if some guy is riding by with a wifi holochain node on his bicycle and a QR code to connect to it. Your ui will pass it in ultimately to lib3h and need to know that particular "Bootstrap" message was a success or not, so will need a "BootstrapResult".